src/probability/hurdUtils.sig

```diff
@@ -368,9 +368,10 @@ sig
   val ASM_MATCH_MP_TAC_N : int -> thm list -> tactic
   val ASM_MATCH_MP_TAC : thm list -> tactic
 
-  (* Tacticals for better expressivity, added by Chun Tian *)
-  val fix                       : Q.tmquote list -> tactic
-  val set                       : Q.tmquote list -> tactic
-  val take                      : Q.tmquote list -> tactic
+  (* Tacticals for better expressivity *)
```

Comment:
Please consider moving these to `bossLib` and documenting them with `.doc` files.  Note that we already have `qx_genl_tac`, but it would be nice to also have `qexistsl_tac` or similar

Reply:
I'm adding `qexistsl_tac` and `qunabbrevl_tac` into `bossLib` and removing all uses of `fix`, `set` (never used), `unset` and `take` from probability scripts, then also removing them from `hurdUtils`.